eclipse-scout-core/src/tile/fields/tablefield/TileTableField.less

```diff
@@ -53,6 +53,27 @@
       & > .table-header-item {
         color: @tile-table-default-color;
 
+        &.sorted,
+        &.grouped,
+        &.filtered {
+          padding-right: @table-header-item-padding-right;
+
+          &.halign-right,
+          &.halign-center {
+            padding-right: @table-cell-padding-left;
+          }
+        }
+
+        &.sorted.grouped,
+        &.sorted.filtered {
+          padding-right: @table-header-item-padding-right;
+
+          &.halign-right,
+          &.halign-center {
+            padding-right: @table-cell-padding-left;
```

Comment:
The header text and the cell content are not aligned correctly for a right aligned column if it is not the last column.

Proposal:

```
&.sorted,
&.grouped,
&.filtered {
  padding-right: @table-header-item-padding-right;

  &.halign-right,
  &.halign-center {
    padding-right: @table-header-item-padding-left;
  }

  &.last.halign-left,
  &.last.halign-center,
  &.last.halign-right {
    padding-right: @table-header-item-padding-right-last;
  }
}
```